### PR TITLE
Support node service

### LINKED ISSUE #7: Escualo should support node services

### PR DESCRIPTION
**Pull request by faloi**
Closes #7

### CODE REVIEW

File: service [hunk 1]
```diff
@@ -11,6 +11,21 @@ source $(dirname $0)/version
...
+      ENV_VARIABLES="";
+      INSTALL_COMMAND="bundle install";
+      LAUNCH_COMMAND="exec bundle exec rackup -o 0.0.0.0 -p $PORT";
+      break;;
+    NodeJS )
+      ENV_VARIABLES="env PORT=$PORT";
+      INSTALL_COMMAND=". /root/.nvm/nvm.sh && npm install --unsafe-perm";
+      LAUNCH_COMMAND="exec bash -c 'source /root/.nvm/nvm.sh && npm start'";
+      break;;
+  esac
```
Comment (flbulgarelli) [inline service]: Perhaps in the future we could just detect the environment from the git repository files, like Heroku does.

Comment (flbulgarelli) [inline service]: It is ok by now

Comment (faloi) [inline service]: Sure, I thought about that. But won't do it while we maintain this _inceptionish_ Bash, I'd like to translate it to Ruby first.

Comment (flbulgarelli) [inline service]: Makes sense

**Pull request merged**